Source/non_ip/ai.conf

```diff
@@ -53,3 +53,10 @@ DOMAIN-SUFFIX,clipdrop.co
 DOMAIN-SUFFIX,jasper.ai
 # OpenArt
 DOMAIN-SUFFIX,openart.ai
+# GitHub Copilot
+DOMAIN-SUFFIX,githubcopilot.com
+DOMAIN,copilot-proxy.githubusercontent.com
+# Github Copilot (Claude)
+DOMAIN,copilot-telemetry.githubusercontent.com
+DOMAIN-SUFFIX,exp-tas.com
```

Comment:
Nope, this is the VSCode AB Test domain. It has nothing to do with GitHub Copilot.

Reply:
In fact GitHub Copilot can't be done without the ide plugin, and the github copilot plugin calls vscode's telemetry interface to determine the region (maybe).